src/olympia/constants/base.py

```diff
@@ -432,46 +414,46 @@
 # Note: Don't change these since they're used as keys in the database.
 REVIEWED_MANUAL = 0
 REVIEWED_ADDON_FULL = 10
-REVIEWED_ADDON_PRELIM = 11
+_REVIEWED_ADDON_PRELIM = 11
 REVIEWED_ADDON_UPDATE = 12
 REVIEWED_DICT_FULL = 20
-REVIEWED_DICT_PRELIM = 21
+_REVIEWED_DICT_PRELIM = 21
 REVIEWED_DICT_UPDATE = 22
 REVIEWED_LP_FULL = 30
-REVIEWED_LP_PRELIM = 31
+_REVIEWED_LP_PRELIM = 31
 REVIEWED_LP_UPDATE = 32
 REVIEWED_OVERDUE_BONUS = 2
 REVIEWED_OVERDUE_LIMIT = 7
 REVIEWED_PERSONA = 40
 # TODO: Leaving room for persona points based on queue.
 REVIEWED_SEARCH_FULL = 50
-REVIEWED_SEARCH_PRELIM = 51
+_REVIEWED_SEARCH_PRELIM = 51
 REVIEWED_SEARCH_UPDATE = 52
 REVIEWED_THEME_FULL = 60
-REVIEWED_THEME_PRELIM = 61
+_REVIEWED_THEME_PRELIM = 61
```

Comment:
Added in https://github.com/mozilla/addons-server/pull/3643/commits/94b263f4c0edf9bac6a582d61662ddc41dcefabb